src/NativeScript/inspector/GlobalObjectInspectorController.cpp

```diff
@@ -125,7 +125,19 @@ GlobalObjectInspectorController::~GlobalObjectInspectorController() {
 }
 
 void GlobalObjectInspectorController::registerDomainDispatcher(WTF::String domainIdentifier, JSC::JSCell* constructorFunction) {
-    std::unique_ptr<DomainInspectorAgent> domainInspectorAgent = std::make_unique<DomainInspectorAgent>(m_globalObject, domainIdentifier, constructorFunction);
+    AgentContext baseContext = {
```

Comment:
I wanted to make that a class field but couldn't initialize it in the constructor. I am open for suggestions on how to do it :) 

```{ { *this, *m_injectedScriptManager, m_frontendRouter.get(), m_backendDispatcher.get() }, globalObject }``` fails with
```InspectorAgentBase.h:51:5: Candidate constructor not viable: cannot convert initializer list argument to 'Inspector::AgentContext &'```